src/editor/components/components/ActionBar/ActionBar.component.jsx

```diff
@@ -0,0 +1,34 @@
+import { Circle20Icon, ArrowDown24Icon } from '../../../icons';
+import Events from '../../../lib/Events';
+
+import styles from './ActionBar.module.scss';
+
+const ActionBar = ({ handleAddClick, isAddLayerPanelOpen }) => {
+  const handleHandClick = () => {
+    Events.emit('hidecursor');
+  };
+
+  const handleSelectClick = () => {
+    Events.emit('showcursor');
+  };
+
+  return (
+    <div>
+      {!isAddLayerPanelOpen && (
+        <div className={styles.wrapper} id="findme">
```

Comment:
Why findeme? Don't set unnecessary id if you don't use it.

Reply:
this is all wip. was for debugging purposes